sound/soc/sof/sof-audio.h

```diff
@@ -23,6 +23,8 @@
 
 #define SOF_AUDIO_PCM_DRV_NAME	"sof-audio-component"
 
+#define SOF_WIDGET_MAX_NUM_PINS	8
```

Comment:
please add a comment that the existing IPC4 firmware only support 8 src or sink queues per widget, hence the limitation. This is useful to explain/document how you chose this value.